PSKoans/Koans/IntroTraining/AboutBooleans.koans.ps1

```diff
@@ -0,0 +1,71 @@
+using module PSKoans
+[Koan(Position = 108)]
+param()
+<#
+    Booleans
+
+    Booleans in PowerShell are either True or False, 1 or 0, on or off.
+
+    They allow us to represent a logical, binary, state of being. 
+
+    "Can I stop this service?" The answer is yes or no.
+
+    Therefore, the CanStop property for a System.ServiceProcess.ServiceController object
+    is a Boolean, shorthanded to bool. As the only possible respone we expect to the question
+    "Can i stop this service?" is a yes or no answer.
+
+    Check for yourself by running the following code:
+
+    Get-Service | Get-Member
+
+    Then scroll down till you see the "CanStop" property:
+
+    CanStop                   Property      bool CanStop {get;}
+
+    Bools are either:
+    
+    $True
+
+    Or:
+
+    $False
+
+#>
+
+Describe "Booleans" {
+<#
+
+    Using only booleans, so $True or $False, answer the following questions
+
+#>
+
+    It "True or False" {
+
+        # Is one greater than two? 
+        # Replace "__" with $True or $False
+        "__" -eq (1 -gt 2) | should -be $true -Because "1 is greater than two"
```

Comment:
I think these can be simplified. PowerShell's syntax is generally sufficiently verbose enough to make this very readable even without many comments. Can we write all of these like this?

```powershell
# Replace the blanks with $true or $false
$____ | Should -Be (1 -gt 2) -Because '1 is not greater than 2'
```

The first one can use a comment prompt like above and then we don't need to repeat it beyond that; the rest can probably go without any comments, really. I'm not sure that restating the assertion in a comment adds a lot of value; again, maybe just on the first one, and then after that the pattern is the same, so I think it's reasonable for users to infer the question being asked without it needing explicit comments. 🙂

Reply:
Happy to take out the ```Replace``` and ```e.g.``` comments after the first time like we did with AboutBinary.

Also quite happy to simplify the ```Should``` bits:
```PowerShell
$____ | Should -Be (1 -gt 2) -Because '1 is not greater than 2'
```

Regarding the comments, I'd personally say that the comparison operators should be explained in this context as the comparison operators koan isn't covered till later on? As a middle-ground why don't we have comments for each time a new comparison operator is used, but then not afterwards? So the first time ```-eq``` is used it's explained it means equals, but afterwards users are expected to infer the question(s) being asked?

Either way, the ```Should``` items should be changed in the next commit ⌨️

Reply:
Actually, re-reading it whilst I'm editing the ```Because``` bits do explain the question so I'll take those comments out.